terraform.tf.j2

```diff
@@ -12,7 +12,7 @@ terraform {
 
   required_providers {
     aws = {
-      version = "~> 4.30.0"
+      version = "~> 3.42.0"
     }
```

Comment:
why downgrade the provider?

Reply:
Good point - I think it's fine to upgrade, just wondered if the 4.30.0 version was causing issues